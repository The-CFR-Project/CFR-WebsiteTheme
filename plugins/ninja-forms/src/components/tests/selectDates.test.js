import { render, cleanup } from '@testing-library/react';
import { globalParams } from './mockData';
import { SelectDates } from '../';

describe( 'submissions-select-dates', () => {
	afterEach( cleanup );
	it( 'Matches snapshot', () => {
		const dateFormat = globalParams.dateFormat;
		const timeFormat = globalParams.timeFormat;
		const setDates = jest.fn();

		const { container } = render(
			<SelectDates
				setDates={ setDates }
				dateFormat={ dateFormat }
				timeFormat={ timeFormat }
			/>
		);
		expect( container ).toMatchSnapshot();
	} );
} );
