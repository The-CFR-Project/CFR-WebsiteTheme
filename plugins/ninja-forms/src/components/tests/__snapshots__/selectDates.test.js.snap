// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-select-dates Matches snapshot 1`] = `
<div>
  <input
    class="flatpickr-input"
    data-mode="range"
    readonly="readonly"
    type="text"
    value=""
  />
</div>
`;
